concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -3325,3 +3349,139 @@ jobs:
               - |
                 paas-cf/concourse/scripts/check-vars-store-certificates-email.sh \
                   "${DEPLOY_ENV}" "${SYSTEM_DNS_ZONE_NAME}" "${ALERT_EMAIL_ADDRESS}"
+
+  - name: health-snapshot
+    serial: true
+    build_logs_to_retain: 25
+    plan:
+      - get: health-snapshot-timer
+        trigger: true
+      - get: bosh-secrets
+      - get: bosh-CA-crt
+      - get: vpc-tfstate
+      - task: generate-snapshot-json
+        timeout: 3m
+        params:
+          BOSH_ENVIRONMENT: ((bosh_fqdn))
+          BOSH_CA_CERT: bosh-CA-crt/bosh-CA.crt
+          BOSH_DEPLOYMENT: ((deploy_env))
+          AWS_DEFAULT_REGION: ((aws_region))
+        config:
+          platform: linux
+          image_resource:
+            type: docker-image
+            source:
+              repository: governmentpaas/awscli
+              tag: b2495d6ed07f680125d19aa7d1701da7efabb289
+          inputs:
+            - name: bosh-secrets
+            - name: bosh-CA-crt
+            - name: vpc-tfstate
+          outputs:
+            - name: health-snapshot-data
+          run:
+            path: sh
+            args:
+              - -e
+              - -u
+              - -c
+              - |
```

Comment:
Is it worth moving to a separate file? My pipeline diff looks terrible - I cannot sanity check easily without viewing the source.

Reply:
Hmmm ... if the script was moved into a file within paas-cf then that would mean this job would have to have a dependency on the paas-cf resource which means it would have to check it out each time just to run the script which seems like overkill.... is that what you are suggesting or am I missing something in concourse that allows you to break things out into separate files without storing it in a resource?

Reply:
This is indeed what I was suggesting, however I'm not in the position to block the delivery on that.